paper-webserver.tex

```diff
@@ -65,14 +65,14 @@ \section{Introduction}
+The rapid characterization and comparison of bacterial pathogens relies principally on the combination of outputs from multiple software programs that are targeted for specific applications. Examples include the identification of known antimircrobial (AMR) resistance genes, through software such as the Resistance Gene Identifier (RGI) \cite{mcarthur2013comprehensive}, {ResFinder}, {ARG-ANNOT}, and {ARIBA}; the identification of known virulence genes through software such as VirulenceFinder \cite{kleinheinz2014applying}, {SRST2}, and {an additional reference}; the annotation of bacterial genomes, through software such as Prokka \cite{doi:10.1093/bioinformatics/btu153}, {another}, and {another}. \todo{Add a section on serotyping and subtyping a la Phylotyper}. These methods represent \textit{in-silico} analogues of traditional wet-lab tests, which allows new WGS results to be viewed in the context of historical tests, and greatly expedites the analyses of newly sequenced genomes. 
 
 Comprehensive platforms that combine individual programs into a cohesive whole also exist. These include free platforms such as the Bacterium Analysis Pipeline (BAP) \cite{thomsen2016bacterial}, the Integrated Rapid Infectious Disease Analysis (IRIDA) project \url{http://www.irida.ca/}, and the Pathosystems Resource Integration Center (PATRIC) \cite{wattam2013patric}. Commercial applications, such as Bionumerics, which is used by PulseNet International for the analyses of WGS data in outbreak situations also exist, and offer support as well as accredited, standardized tests \cite{swaminathan2001pulsenet}.
 
 % 3. 100,000 genomes, (lookup number for Enterobase, GenBank), how do we run analysis on it
-Traditionally, these platforms have been applied to samples numbering only a few dozen while WGS data for bacterial pathogens of public health importance have recently accumulated in public databases in the tens of thousands, with hundreds of thousands set to be available within the next few years. For \textit{Escherichia coli}, there are over sixty thousand publicly available genomes in EnteroBase \url{https://enterobase.warwick.ac.uk/} and three million whole genomes in GenBank \cite{doi:10.1093/nar/gks1195}.
+Traditionally, these platforms have been applied to samples numbering only a few dozen \cite{} while WGS of bacterial pathogens have recently accumulated in public databases in the hundreds of thousands, with millions set to be available within the next few years. For \textit{Escherichia coli} alone, there are over sixty thousand publicly available genomes in EnteroBase \url{https://enterobase.warwick.ac.uk/} and three million sequenced genomes in GenBank \cite{doi:10.1093/nar/gks1195}.
```

Comment:
`the number of publicly available whole genome sequences is currently in the hundreds of thousands` 2 places